front/src/store/tv/actions.ts

```diff
@@ -85,8 +85,6 @@ export const requestBasicShowInfoAction = (): AppThunk => async (dispatch, getSt
     validCachedIds.forEach(id => {
       combinedData[id] = cachedBasicShowInfo[id];
     });
-
-  // Fetch data for ids that are not cached and add to combinedData
```

Comment:
why was this deleted?

Reply:
sorry I did it by mistake. I was trying to understand how this action works and had my own comments added and then deleted. Regarding testing more components. Yep we can start testing there as well. Everything should be tested.